.travis.yml

```diff
@@ -16,12 +18,3 @@ env:
   - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome
   - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome ALWAYS_LOCATE=false
   - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome PREFER_CSS=1 SELECTOR_STATS=1
-  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs
-  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs ALWAYS_LOCATE=false
-  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs PREFER_CSS=1 SELECTOR_STATS=1
-  - RAKE_TASK=yard:doctest
```

Comment:
Oh, that was a mistake. Delete happy mood I guess. :)
On Oct 5, 2015 12:26 AM, "Alex Rodionov" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/watir/watir-webdriver/pull/358#discussion_r41111356>:
>
> > @@ -16,12 +18,3 @@ env:
> >    - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome
> >    - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome ALWAYS_LOCATE=false
> >    - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome PREFER_CSS=1 SELECTOR_STATS=1
> > -  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs
> > -  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs ALWAYS_LOCATE=false
> > -  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs PREFER_CSS=1 SELECTOR_STATS=1
> > -  - RAKE_TASK=yard:doctest
>
> You remove doctests :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/watir/watir-webdriver/pull/358/files#r41111356>.
>